### PR TITLE
Improve non-PHP HTML generation

### PR DESCRIPTION
**Pull request by nwf**
Now includes all the PNG tabs from the TeX handbook as data URIs of
their PNG forms.  How do you like that?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,3 +27,10 @@ presentation.pdf: presentation/presentation.tex
 	cd presentation && pdflatex presentation.tex
 	cp presentation/presentation.pdf .
 
+fullplain.html: rules/generate.rb rules/full.txt
+fullplain.html: $(patsubst %.svg,%.png.b64,$(wildcard rules/texfiles/*.svg))
+fullplain.html:
+	cd rules && ruby generate.rb -T ../fullplain.html
```
Comment (timparenti) [inline Makefile]: Given that [the new `ctfwsd` for the HTML handbook](https://github.com/cmukgb/ctfws-host-daemons/blob/master/inotifywait-handbook) will be polling `handbook.html` (by extension from `handbook.pdf`), do we want that to be the target here instead?

File: Makefile [hunk 2]
```diff
@@ -27,3 +27,10 @@ presentation.pdf: presentation/presentation.tex
...
+
+rules/texfiles/%.png.b64 : rules/texfiles/%.svg
+	convert -density 1200 $< png:- | base64 -w 0 > $@
```
Comment (timparenti) [inline Makefile]: `convert` requires `imagemagick`, which I've installed on the new server and will add to the README as a requirement in another commit.

It seems that all the resultant `*.png.b64` files are empty, though.

**Review by timparenti — CHANGES_REQUESTED** [review]
I'm not sure whether cross-references ever worked with the `-T` option on `generate.rb`, but the resultant HTML has a lot of `(see )` scattered throughout, with no description or link of what cross-reference should be followed.

File: Makefile (hunk 1 above)
Comment (nwf) [reply Makefile]: That seems like a good plan.

File: Makefile (hunk 2 above)
Comment (nwf) [reply Makefile]: I.. am very curious why that is.  I wonder if this is "the server's version of imagemagick is ancient".
I've tried to go a less clever, more disk-using, path of generating a .png and a .png.b64 separately; please advise.

Comment (timparenti) [reply Makefile]: The "new" server is currently running `ImageMagick 6.8.9-9 Q16 x86_64 2016-11-26`.

**Comment by nwf** [comment]
Found the "(see )" bug and fixed that, too.

**Review by timparenti — APPROVED** [review]

**Pull request merged**